source/nvda.pyw

```diff
@@ -89,6 +89,13 @@ parser.add_argument('-k','--check-running',action="store_true",dest='check_runni
 parser.add_argument('-f','--log-file',dest='logFileName',type=str,help="The file where log messages should be written to")
 parser.add_argument('-l','--log-level',dest='logLevel',type=int,default=0,choices=[10, 12, 15, 20, 30, 40, 50, 100],help="The lowest level of message logged (debug 10, input/output 12, debugwarning 15, info 20, warning 30, error 40, critical 50, off 100), default is info")
 parser.add_argument('-c','--config-path',dest='configPath',default=None,type=str,help="The path where all settings for NVDA are stored")
+parser.add_argument(
+	'--lang',
+	dest='cmdLineLanguage',
+	default=None,
+	type=str,

```

Comment:
Instead of str, you can provide a function that does further parsing and normalization of the name. See the stringToBool function as an example. Could you also have a look at locale.normalize? I think that does pretty much what you're doing manually now. If it is not sufficient, I think I'd like to see a comment why not.

Reply:
> Instead of str, you can provide a function that does further parsing and normalization of the name. See the stringToBool function as an example.

I'll have a look.

> Could you also have a look at locale.normalize? I think that does pretty much what you're doing manually now. If it is not sufficient, I think I'd like to see a comment why not.

```
>>>locale.normalize("en")
'en_US.ISO8859-1'
```
This is not the kind of output we need.

`languageHandler.normalizeLanguage` does not fit either as it requires a dialect to be specified.